examples/security/secrets/secret-setup/src/main.rs

```diff
@@ -0,0 +1,50 @@
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // put secrets in secret store
+    let component_secret = wasmcloud_secrets_types::Secret {
+        name: "api_password".to_string(),
+        string_secret: Some("opensesame".to_string()),
+        ..Default::default()
+    };
+    let provider_secret = wasmcloud_secrets_types::Secret {
+        name: "redis_password".to_string(),
+        string_secret: Some("sup3rS3cr3tP4ssw0rd".to_string()),
+        ..Default::default()
+    };
```

Comment:
In lieu of some top level docs describing this binary, what do you think about moving these to where they're used? 

Then this `main` would be like... 4 lines, which are really clearly putting secrets.

Also, if you made the function `put_secrets` take an iterator of secrets we could do multiple at the same time!

Reply:
> what do you think about moving these to where they're used?

What do you mean by this, like moving the secret out to another place?

Reply:
Ah this was a NIT about the secret creation itself -- they could have been defined inline. Definitel ya NIT though!

Reply:
Ah okay. Yeah I think it looks a little cleaner now with the moved function to secrets-nats-kv, let me know what you think. We can also add an `add_mapping` function now that I'm thinking about it to make it more clear

Reply:
Oh there are a couple more things we could do here -- we could make the fields `wasmcloud_secrets_types::Secret` *not* `pub` so that we can enforce the string/binary secret difference (or use an enum), and some constructors would be nice... but honestly that's all icing.